amoro-ams/amoro-ams-dashboard/src/views/tables/components/Selector.vue

```diff
@@ -33,8 +33,14 @@ const tagSearchKey = ref<string>('')
 const tabActiveKey = ref<string>(branchTypeMap.BRANCH)
 const branchList = ref<IBranchItem[]>([])
 const tagList = ref<IBranchItem[]>([])
-const actualBranchList = computed(() => branchList.value.filter(item => !branchSearchKey.value || item.label.includes(branchSearchKey.value)))
-const actualTagList = computed(() => tagList.value.filter(item => !tagSearchKey.value || item.label.includes(tagSearchKey.value)))
+const consumerList = ref<IBranchItem[]>([])
+const actualBranchList = computed(() =>branchList.value.filter((item) =>!branchSearchKey.value ||
+ item.label.includes(branchSearchKey.value)))
+const actualTagList = computed(() =>tagList.value.filter((item) => !tagSearchKey.value || 
+item.label.includes(tagSearchKey.value)))
+const actualConsumerList = computed(() =>consumerList.value.filter((item) => !tagSearchKey.value || 
```

Comment:
Can we define a consumerSearchKey instead?

Reply:
Of course, I will fix these bugs